perf/istio-install/base/templates/prometheus-operator.yaml

```diff
@@ -184,4 +184,171 @@ spec:
   selector:
     app.kubernetes.io/component: controller
     app.kubernetes.io/name: prometheus-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: example
+    role: alert-rules
+  name: prometheus-example-rules
+  namespace: istio-prometheus
+spec:
+  groups:
+    - name: ./example.rules
+      rules:
+        - alert: ExampleAlert
```

Comment:
After this is in place, do we have a set of intended alerts defined, or is defining them the next step?

Reply:
@douglas-reid The example one just existed for the purpose of testing slack and webhook integration. Now I added some real basic alerting rules under the PrometheusRule CR, can you help check?

Reply:
the rules look ok to me at a first glance. have you validated they perform as expected in use?

Reply:
yep for the ones I added already they work, but I would need to add more rules later